packages/react/src/ReactElementValidator.js

```diff
@@ -281,8 +281,8 @@ export function createElementWithValidation(type, props, children) {
   const validType =
     typeof type === 'string' ||
     typeof type === 'function' ||
-    typeof type === 'symbol' ||
-    typeof type === 'number';
+    typeof type === 'number' ||
+    type === 'REACT_FRAGMENT_TYPE';
```

Comment:
I still don’t understand why this change is necessary.

Reply:
Because type of polyfilled Symbol is object. This check cannot catch this situation.

Reply:
Shouldn’t this compare to REACT_FRAGMENT_TYPE itself then? Without quotes.

Reply:
Oh, thanks. I am so stupid.